### PR TITLE
Add "Other summits" final section to all summit pages

### PR DESCRIPTION
**Pull request by magnusbaeck**
### Applicable Issues
#37 

### Description of the Change
The new section at the end of every summit page makes the other summit pages more visible. Without this you have to look into the Git repository and read the files there or use it construct a URL.

Note that this PR is based on #45 so changes from both PRs will show up in the diff. In the diff view you can choose to only include one of the commits. The PR has been deployed to https://magnusbaeck.github.io/eiffel-community.github.io.

### Alternate Designs
None.

### Benefits
Old material can be found.

### Possible Drawbacks
None.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I    have the right to submit it under the open source license    indicated in the file; or

(b) The contribution is based upon previous work that, to the best    of my knowledge, is covered under an appropriate open source    license and I have the right under that license to submit that    work with modifications, whether created in whole or in part    by me, under the same open source license (unless I am    permitted to submit under a different license), as indicated    in the file; or

(c) The contribution was provided directly to me by some other    person who certified (a), (b) or (c) and I have not modified    it.

(d) I understand and agree that this project and the contribution    are public and that a record of the contribution (including all    personal information I submit with it, including my sign-off) is    maintained indefinitely and may be redistributed consistent with    this project or the open source license(s) involved.

Signed-off-by: Magnus Bäck \<magnus.back@axis.com>

### CODE REVIEW

File: summit-2020-1.html [hunk 1]
```diff
@@ -45,6 +45,17 @@ <h1 class="section-heading">Agenda</h1>
...
         </div>
     </section>
-    <div data-include="../includes/footer.html"></div>
+    <section>
+      <div>
+        <h1 class="section-heading">Past summits</h1>
+        <p class="section-paragraph">
+          <ul>
+            <li><a href="summit-2019-2.html">Summit 2019-2</a></li>
+            <li><a href="summit-2021-1.html">Summit 2021-1</a></li>
```
Comment (e-backmark-ericsson) [inline summit-2020-1.html:54]: I guess the 2021 summit should not show up here?
```suggestion
```

File: summit-2019-2.html [hunk 2]
```diff
@@ -110,6 +110,17 @@ <h1 class="section-heading">Register here</h1>
         </p>
       </div>
     </section>
-    <div data-include="../includes/footer.html"></div>
+    <section>
```
Comment (e-backmark-ericsson) [inline summit-2019-2.html:113]: I guess there isn't any need for any "Past summits" section since this is the oldest one we've archived?

**Review by e-backmark-ericsson — CHANGES_REQUESTED** [review]

File: summit-2019-2.html (hunk 2 above)
Comment (magnusbaeck) [reply summit-2019-2.html:113]: My thought was that "past" was relative to today and not the summit in question, i.e. the list would contain all past summits to make them all more accessible.

Comment (magnusbaeck) [reply summit-2019-2.html:113]: Or maybe "Other summits"? Then it makes sense to link to the current summit (i.e. summit.html) which may or may not have appeared in the past.

Listing the summits in a single file that's included from all files is certainly an option, but then all summit pages will have a link to itself. Given the low frequency of summits I figured we could manage to keep that list in sync by hand.

Comment (e-backmark-ericsson) [reply summit-2019-2.html:113]: Agreed. "Other summits" sounds good to me.

Comment (magnusbaeck) [reply summit-2019-2.html:113]: Done. Reviewing commit stacks on GH is a travesty as usual but I merged from the parent branch into this one and added a couple of commits on top.

**Review by e-backmark-ericsson — APPROVED** [review]

**Comment by m-linner-ericsson** [comment]
@magnusbaeck Great work!

**Pull request merged**